src/main/scala/com/redislabs/provider/redis/rdd/RedisRDD.scala

```diff
@@ -94,35 +85,130 @@ class RedisListRDD(prev: RDD[String], val rddType: String) extends RDD[String](p
   def getSET(nodes: Array[RedisNode], keys: Iterator[String]): Iterator[String] = {
     groupKeysByNode(nodes, keys).flatMap {
       x =>
-        {
-          val conn = x._1.endpoint.connect()
-          val setKeys = filterKeysByType(conn, x._2, "set")
-          val res = setKeys.flatMap(conn.smembers).iterator
-          conn.close
-          res
-        }
+      {
+        val conn = x._1.endpoint.connect()
+        val setKeys = filterKeysByType(conn, x._2, "set")
+        val res = setKeys.flatMap(conn.smembers).iterator
+        conn.close
+        res
+      }
     }.iterator
   }
   def getLIST(nodes: Array[RedisNode], keys: Iterator[String]): Iterator[String] = {
     groupKeysByNode(nodes, keys).flatMap {
       x =>
-        {
-          val conn = x._1.endpoint.connect()
-          val listKeys = filterKeysByType(conn, x._2, "list")
-          val res = listKeys.flatMap(conn.lrange(_, 0, -1)).iterator
-          conn.close
-          res
-        }
+      {
+        val conn = x._1.endpoint.connect()
+        val listKeys = filterKeysByType(conn, x._2, "list")
+        val res = listKeys.flatMap(conn.lrange(_, 0, -1)).iterator
+        conn.close
+        res
+      }
     }.iterator
   }
 }
 
+class RedisZSetConf() extends Serializable {
```

Comment:
I think a better name would be ZSetContext, it's not really a configuration, but an execution context